js/experiment.js

```diff
@@ -0,0 +1,14 @@
+function drawRect() {
+	var c = document.getElementById("theCanvas");
+	var ctx = c.getContext("2d");
+	ctx.fillStyle = "#FF00FF";
```

Comment:
I've created JavaScript code and documentation etiquette guides in the Courseography Wiki in order to make the code share the same features across the code base. These are small things like the use of the single quote `'` over the double quote `"` and JSDocs for function documentation. If you're able to make these changes, I would greatly appreciate it! :smiley_cat: